dataactbroker/handlers/accountHandler.py

```diff
@@ -387,6 +387,10 @@ def updateUser(self, system_email):
         if requestDict.exists("is_active"):
             is_active = bool(requestDict.getValue("is_active"))
             if not self.isUserActive(user) and is_active:
+                # Reset password count to 0
+                self.resetPasswordCount(user)
+                # Reset last login date so the account isn't expired
+                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
```

Comment:
Oh right understanding that now, but we're still putting a false piece of info in here, this user has logged in before, what happens if we then want to know when they did last log in?